gamestonk_terminal/prediction_techniques/fbprophet.py

```diff
@@ -95,6 +95,9 @@ def fbprophet(l_args, s_ticker, df_stock):
 
         df_stock = df_stock.sort_index(ascending=True)
         df_stock.reset_index(level=0, inplace=True)
+        df_stock.columns = map(
+            str.lower, df_stock.columns
```

Comment:
I think https://github.com/DidierRLopes/GamestonkTerminal/pull/155 fixed this. This should be no longer necessary.